lib/resty/core/shdict.lua

```diff
@@ -404,6 +454,8 @@ if ngx_shared then
                 mt.replace = shdict_replace
                 mt.delete = shdict_delete
                 mt.flush_all = shdict_flush_all
+                mt.get_pool_slot_count = shdict_get_pool_slot_count
+                mt.get_stats = shdict_get_stats
```

Comment:
Let's rename this method to just `stats()`?

Reply:
OK, done.